.github/workflows/pre-commit.yml

```diff
@@ -0,0 +1,22 @@
+name: pre-commit
+
+on: [pull_request]
```

Comment:
Do we need to add what branches this applies to like the previous action (lines 6-8)?

Reply:
I'm not sure, if we are going to keep the pre-commit action read-only then yes. Otherwise, if it will have the ability to modify the commit I think it is better not to have it on push events since it might be unexpected.

Reply:
On second thoughts, since now it is read-only it's okay to have it on push events too. We will think about it when and if it will be given the ability to modify the commit.